buildSrc/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-testing.gradle.kts

```diff
@@ -6,8 +6,8 @@ plugins {
   id("net.bytebuddy.byte-buddy")
 
   id("io.opentelemetry.instrumentation.base")
-  id("io.opentelemetry.instrumentation.javaagent-codegen")
   id("io.opentelemetry.instrumentation.javaagent-shadowing")
+  id("io.opentelemetry.instrumentation.javaagent-codegen")
```

Comment:
Did you have an ordering issue? We can fix it later, but good not to have ordering constraints so just checking

Reply:
No issues. I first removed that but it turned out still required so I put it back.